test/verify.test.js

```diff
@@ -509,7 +509,7 @@ test.serial('Throw error if GitLab API return any other errors', async t => {
   const env = {GITLAB_TOKEN: 'gitlab_token'};
   const gitlab = authenticate(env)
     .get(`/projects/${owner}%2F${repo}`)
-    .times(3)
+    .times(1)
```

Comment:
This was changed intentionally so as to not bloat the diffs with additional logic to facilitate what I can only assume was default retry logic contains in the `got` module.